presto-docs/src/main/sphinx/release/release-0.286.rst

```diff
@@ -0,0 +1,88 @@
+=============
+Release 0.286
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug for min_by/max_by for window function, where results are incorrect when the function specifies number of elements to keep and the window does not have “unbounded following” in the frame.
+* Fix a potential bug in EXCEPT and INTERSECT queries. Do not prune unreferenced output in intersect and except nodes. :pr:`21343`
+* Fix bug of pull lambda out optimizers when expressions pulled out are of type JoniRegexType or LikePatternType.
+* Fix lambda expression pull out optimizer so that it will be able to extract independent expressions from conditional expression. :pr:`21344`
+* Fix serialized name for retriable QueryError property.
+* Fix the compilation error of merge function for KHyperLogLog state.
+* Fixed a bug in cte materialization which was causing incorrect plan generation in some cases.
+* Fixed redis-provider-plugin docs which previously contained an incorrect file name for property loading, and included the HBO configurations for the coordinator.
+* Fixed writtenIntermediateBytes metric by ensuring its correct population through temporary table writers.
+* Improve efficiency of partial aggregation step. This can be enabled by setting the ``adaptive_partial_aggregation`` session property to true.
```

Comment:
```suggestion
* Add support for adaptive partial aggregation which disables partial aggregation in cases where it could be inefficient. This feature is configurable by the session property ``adaptive_partial_aggregation`` (disabled by default)
```